### DISCUSSION TITLE
Activity overview

### DISCUSSION THREAD

**Discussion by Random-user420** [community/community#143455] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, my Activity overview isn't showing all commits I made. I guess that are only the commits and pull requests I did on the github web page. Here is a screenshot of the chart: (https://github.com/Random-user420/bestKabu)
![image](https://github.com/user-attachments/assets/6b1f3129-10ab-4a68-9313-131a0e69c1b8)
And as an exaple I did on this one Repo 120 commits in the last months: (https://github.com/Random-user420/bestKabu/graphs/contributors)
![image](https://github.com/user-attachments/assets/1eb23100-75a2-4fae-b9a8-0535411dfb7b)

My git e-mail is the same as my github, and the commits show up as mine on that repo, so what am I doing wrong?

**Comment by mgunawardhana** [comment, marked as answer]
1. **Commit Email Configuration**: GitHub contributions are associated with the email configured in the commits. Even though they mentioned that their Git email matches their GitHub email, it's worth double-checking:
   - Ensure that the primary email in GitHub settings matches the email in their local Git configuration (`git config user.email`).
   - Make sure the email address used in commits is verified on GitHub.

2. **Repository Visibility**: Contributions made to private repositories do not show up in the public contribution graph unless the user has enabled showing private contributions. To enable this:
   - Go to **GitHub profile settings > Profile settings > Contributions & activity** and make sure **"Include private contributions on my profile"** is checked.

3. **Contribution Rules on GitHub**: GitHub only counts certain activities as contributions:
   - Committing to the default branch (often `main` or `master`) or the repository's designated primary branch.
   - Opening issues or pull requests in public repositories.
   - Being a collaborator or contributor on the repository.
   - **Note**: If the user is committing to a feature branch that’s not merged into the default branch, these commits may not count.

4. **Checking Contribution Types**: If they're using forks or have commits not on the primary branch, those might not show up. They should check if their work was done on a fork and, if so, if they've made pull requests to the original repo's primary branch.

5. **Contribution Chart Delay**: Sometimes, GitHub's contribution graph can take a bit of time to update. If they recently made a significant number of contributions, it may take some time before all updates are reflected.

### Suggested Steps to Resolve
1. Verify email configuration in both Git and GitHub.
2. Enable private contributions if needed.
3. Ensure contributions are made to the default branch or merged into it.
4. Wait a few hours to see if recent contributions reflect on the profile.

if your issue was fixed make this as a answer if it is not we can find out more ways to fix this.

**Reply by Random-user420** [reply]
Thank you for the reply.

My problem is that have a fork of a repo to that I'm commiting to. And the og repo maintainer is no longer active.
